formats/sd-x86_64.nix

```diff
@@ -0,0 +1,13 @@
+{
+  config,
+  lib,
+  pkgs,
+  modulesPath,
+  ...
+}: {
+  imports = [
+    "${toString modulesPath}/installer/sd-card/sd-image.nix"
```

Comment:
There is also:

sd-image-x86_64.nix

in nixpkgs. Should this be used instead?

Reply:
Ok. This was used in the second commit instead?